templates/components/api-server/template/server/middleware.json

```diff
@@ -10,6 +10,26 @@
         "credentials": true,
         "maxAge": 86400
       }
+    },
+    "helmet#xssFilter": {
+      "params": {
+        "setOnOldIE": true
+      }
+    },
+    "helmet#frameguard": {
+      "params": "deny"
+    },
+    "helmet#hsts": {
+      "params": {
+        "maxAge": 0,
+        "includeSubdomains": true
+      }
+    },
```

Comment:
Ha, I didn't realise "hsts" sends the header only for HTTPs requests. That's great, as it allows us to include the middleware by default :+1: